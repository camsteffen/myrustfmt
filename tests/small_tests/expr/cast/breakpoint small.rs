// test-kind: breakpoint-no-change

fn test() {
    a as usize;
}
