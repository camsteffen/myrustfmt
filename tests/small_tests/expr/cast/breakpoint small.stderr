Max width exceeded at line 1
